examples/weaviate/personalized_description_with_weaviate_and_gemini_api.ipynb

```diff
@@ -0,0 +1,963 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.                "X-PaLM-Api-Key": 'PALM-API-KEY' # Replace with your Gemini API key
</pre><p>Can you use our common auth cell to get the API key from a colab secret? It's the <code>Set up your API key</code> cell you will see in notebooks like <a href="https://github.com/google-gemini/cookbook/blob/main/quickstarts/Function_calling.ipynb" target="_blank">this</a> one.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Added! I'm assuming this notebook with be shown via Colab then since I'm not able to get this to work on vscode.</p>